examples/playground/app/page.tsx

```diff
@@ -3,8 +3,26 @@ import { Heading, Text } from "@telegraph/typography";
 export default function Home() {
   return (
     <main>
-      <Heading />
-      <Text/>
+      <Heading size="9">Heading</Heading>
+      <Heading size="8">Heading</Heading>
+      <Heading size="7">Heading</Heading>
+      <Heading size="6">Heading</Heading>
+      <Heading size="5">Heading</Heading>
+      <Heading size="4">Heading</Heading>
+      <Heading size="3">Heading</Heading>
+      <Heading size="2">Heading</Heading>
+      <Heading size="1">Heading</Heading>
+      <Heading size="0">Heading</Heading>
```

Comment:
Few qs:

1. Should size be a string over an integer?
2. Is this what we've agreed upon with Krisna to describe these? I assume so, but asking as last time I thought we had some token names

Reply:
1. I could make it either but typing `size="2"` is a lot more satisfying than typing `size={2}`
3. Yes this matches the figma doc here: https://www.figma.com/file/br4Lk9C5qw3UVyGsAbONDY/Telegraph-design-system?node-id=38%3A327&mode=dev